### PR TITLE
fix: node-forge added setAttributes to the CertificateRequest interface defination

### PR DESCRIPTION
**Pull request by krnbr**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>

### CODE REVIEW

**Comment by krnbr** [comment]
Was getting the following error while using the setAttributes method of the CSR

TS2551: Property 'setAttributes' does not exist on type 'CertificateRequest'

**Comment by krnbr** [comment]
@joeflateau if you can please approve this

File: types/node-forge/index.d.ts [hunk 1]
```diff
@@ -425,6 +425,8 @@ declare module "node-forge" {
              * @return the attribute.
              */
             getAttribute(opts: string | GetAttributeOpts): Attribute | null;
+
+            setAttributes(attrs: any[]): void;
```
Comment (joeflateau) [inline types/node-forge/index.d.ts:429]: we must use a more specific type here than `any`

File: types/node-forge/node-forge-tests.ts [hunk 2]
```diff
@@ -20,6 +20,19 @@ let certPem = forge.pki.certificateToPem(cert);
 let csr = forge.pki.createCertificationRequest();
 csr.publicKey = keypair.publicKey;
 csr.sign(keypair.privateKey);
+csr.setAttributes([
```
Comment (joeflateau) [inline types/node-forge/node-forge-tests.ts:23]: please also test the example you provided in your link:

```
csr.setAttributes([{
  name: 'challengePassword',
  value: 'password'
}, {
  name: 'unstructuredName',
  value: 'My company'
}]);
```

**Review by joeflateau — CHANGES_REQUESTED** [review]

**Pull request closed without merging**